src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/JsonTest/JsonBulkCopyTest.cs

```diff
@@ -2,35 +2,53 @@
 using System.Collections.Generic;
 using System.Data;
 using System.IO;
-using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
 using Newtonsoft.Json.Linq;
 using Newtonsoft.Json;
 using Xunit.Abstractions;
 using Xunit;
+using System.Collections;
 
 namespace Microsoft.Data.SqlClient.ManualTesting.Tests.SQL.JsonTest
 {
+    public class JsonBulkCopyTestData : IEnumerable<object[]>
+    {
+        public IEnumerator<object[]> GetEnumerator()
+        {
+            yield return new object[] { CommandBehavior.Default, false };
+            yield return new object[] { CommandBehavior.Default, true };
+            yield return new object[] { CommandBehavior.SequentialAccess, false };
+            yield return new object[] { CommandBehavior.SequentialAccess, true };
+        }
+        IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
+    }
+
     public class JsonBulkCopyTest
     {
         private readonly ITestOutputHelper _output;
-        private static readonly string _jsonFile = "randomRecords.json";
-        private static readonly string _outputFile = "serverRecords.json";
+        private static readonly string _generatedJsonFile = DataTestUtility.GenerateRandomCharacters("randomRecords");
+        private static readonly string _outputFile = DataTestUtility.GenerateRandomCharacters("serverResults");
+        private static readonly string _sourceTableName = DataTestUtility.GenerateObjectName();
+        private static readonly string _destinationTableName = DataTestUtility.GenerateObjectName();
+        private static readonly int _jsonArrayElements = 300;
+        private static readonly int _rows = 100;
```

Comment:
You could also provide these values in the bulk copy tests as parameters and test different combinations.